client/my-sites/earn/memberships/index.jsx

```diff
@@ -270,12 +270,31 @@ class MembershipsSection extends Component {
 						) }
 					/>
 				) }
+				<SectionHeader label={ this.props.translate( 'About Recurring Payments' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Recurring Payments' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
👎